### PR TITLE
[release] merging master-release-update into master

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
cc @GuillaumeGomez @EPashkin @sdroege

### CODE REVIEW

File: gio/vendor.md [hunk 1]
```diff
@@ -18079,7 +18079,7 @@ a `GAsyncReadyCallback`
 ## `user_data`
 user data for the callback
 <!-- trait SocketListenerExt::fn accept_finish -->
-Finishes an async accept operation. See `SocketListenerExt::accept_async`
+Finishes an async accept operation. See `SocketListener::accept_async`
```
Comment (EPashkin) [inline gio/vendor.md]: It not in `SocketListenerExtManual` ?

Comment (GuillaumeGomez) [reply gio/vendor.md]: Checking.

Comment (GuillaumeGomez) [reply gio/vendor.md]: It is indeed, good catch!

Comment (EPashkin) [reply gio/vendor.md]: Hm, this is local function and it not be in doc, so can be ignored

Comment (GuillaumeGomez) [reply gio/vendor.md]: I wonder if this is a bad generation coming from gir or from somewhere else...

Comment (GuillaumeGomez) [reply gio/vendor.md]: I fixed it anyway. :)

Comment (EPashkin) [reply gio/vendor.md]: But what next text unchanged is bad, gir need fix sometime
```
<!-- trait SocketListenerExt::fn accept_async -->
This is the asynchronous version of `SocketListenerExt::accept`.
```

Comment (GuillaumeGomez) [reply gio/vendor.md]: Damn it.

Comment (EPashkin) [reply gio/vendor.md]: IMHO as we have https://github.com/GuillaumeGomez/rustdoc-stripper/issues/53 too,
better update docs for this version later

Comment (EPashkin) [reply gio/vendor.md]: and this PR can be merged as is but not crate published

Comment (GuillaumeGomez) [reply gio/vendor.md]: Agreed.

**Pull request merged**